Formula/iamy.rb

```diff
@@ -0,0 +1,26 @@
+class Iamy < Formula
+  desc "AWS IAM import and export tool"
+  homepage "https://github.com/99designs/iamy"
+  url "https://github.com/99designs/iamy/archive/v2.0.0.tar.gz"
+  sha256 "44906985ac2a7a0f4a703c4eb872e626770ffa1ea01a589279e75867a4348630"
+  head "https://github.com/99designs/iamy.git"
+
+  depends_on "awscli"
+  depends_on "glide" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/99designs/iamy"
+    dir.install buildpath.children
+    cd dir do
+      system "glide", "install"
+      system "go", "build", "-o", bin/"iamy", "-ldflags", "-X main.Version=v#{version}"
+    end
+  end
+
+  test do
+    system bin/"iamy", "--help"
```

Comment:
I agree. However it is troublesome. The tool has two commands: `pull` and `push`. `pull` connects to an AWS account and writes the current IAM configuration to a YAML representation. Conversely, `push` loads IAM configuration from YAML files and applies it to an AWS account. These operations cannot be exercised without credentials to an active AWS account. Further, I cannot see any automated tests in the source repository.

@ilovezfs What do you suggest in this case?